keyboards/acheron/austin/rules.mk

```diff
@@ -0,0 +1,31 @@
+## chip/board settings
+# the next two should match the directories in
```

Comment:
```suggestion
```

Reply:
Added in https://github.com/qmk/qmk_firmware/pull/7507/commits/7b15b4d5cb0afee90f5213a359a36f14f729ce14